paperqa/clients/__init__.py

```diff
@@ -28,6 +29,7 @@
 
 ALL_CLIENTS: Collection[type[MetadataPostProcessor | MetadataProvider]] = {
     *DEFAULT_CLIENTS,
+    OpenAlexProvider,
```

Comment:
@nadolskit quick question here. If I don't have `OPENALEX_MAILTO`, could I see performance degredations?

Reply:
yeah, I added a quick warning here:
https://github.com/Future-House/paper-qa/pull/555/files#diff-0d2818d1aa2e556d40472804cf93802b8fa28c91c861f9964c994042240dea84R45

it's outlined a bit here:
https://docs.openalex.org/how-to-use-the-api/rate-limits-and-authentication#authentication